### PR TITLE
user context menu

### PR DESCRIPTION
**Pull request by kinjalravel**
## Summary

_A summary of changes being made in this PR_

## Checklist
- [ ] Pre-push checks pass `npm run check`
  - [ ] Linter passing via `npm run lint`
  - [ ] Unit & Integration tests passing via `npm run test:packages`
  - [ ] Docker build process passing via `npm run build-client`
- [ ] If this PR is still a WIP, convert to a draft 
- [ ] When this PR is ready, mark it as "Ready for review"
- [ ] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewers

## References

_References to pertaining issue(s)_

## QA Steps

1. `git checkout pr_branch_name`
2. `npm install`
3. `npm run dev-reinit`
4. `npm run dev`

List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos.

## Reviewers

_Reviewers for this PR_

### CODE REVIEW

File: packages/client-core/src/systems/ui/PersonMenuView.tsx [hunk 1]
```diff
@@ -96,10 +97,27 @@ const AvatarContextMenu = () => {
...
+      const selfId = authState.user.id?.value ?? ''
+      const blockUserId = user.id?.value ?? ''
+      UserService.blockUser(selfId, blockUserId)
+    }
+  }
+
+  const addAsFriend = () => {
+    if (authState.user?.id?.value !== null && user) {
+      const selfId = authState.user.id?.value ?? ''
+      const blockUserId = user.id?.value ?? ''
```
Comment (zulqarnainhanif) [inline packages/client-core/src/systems/ui/PersonMenuView.tsx:116]: please change the name to `friendUserId`

Comment (zulqarnainhanif) [reply packages/client-core/src/systems/ui/PersonMenuView.tsx:116]: or `relatedUserId ` as used in the Service file

Comment (kinjalravel) [reply packages/client-core/src/systems/ui/PersonMenuView.tsx:116]: @zulqarnainhanif Sure I will do that

**Comment by HexaField** [comment]
closing as work is continuing in https://github.com/XRFoundation/XREngine/pull/5034/files

**Pull request closed without merging**